### DISCUSSION TITLE
我在archery中加入了clickhouse实例，检测过程中会把转换为小写

### DISCUSSION THREAD

**Discussion by owqml** [hhyo/Archery#2498] [Q&A]
提交的：insert into xxx_history_data_N20240201 select * from xxx_history_data where toYYYYMMDD(toDateTime(ts / 1000)) = '20240112'

检测报错:  表  xxx_uat.eap_history_data_n20240201 不存在！
我想了解到的是，这是archery进行转换的还是goInception进行的转换。因为clickhouse默认是区分表名大小写的，所以这里对我们使用产生了困扰。

**Comment by LeoQuote** [comment]
可以开一下debug看一下，我这边没有clickhouse的环境 https://github.com/hhyo/Archery/blob/8bec5f4c144362ebac00fa21c30ba50cd75d2014/sql/engines/clickhouse.py#L323 代码基本上是在这里，执行前检查的时候发现表不存在

**Reply by owqml** [reply]
好的，感谢。好像确实是给表明转小写了

**Reply by LeoQuote** [reply]
欢迎pr哈
